magstock/templates/regextra.html

```diff
@@ -220,6 +220,14 @@
     <div class="col-sm-6">
       <input type="text" class="form-control" name="waiver_signature" value="{{ attendee.waiver_signature }}" placeholder="Name exactly as it appears on Photo ID" required />
     </div>
+  </div>
+    <div class="form-group">
+    <label class="col-sm-3 control-label">Date of Signature</label>
+    <div class="col-sm-6">
+      <input type="text" class="form-control" value="{{ now()|datetime_local("%Y-%m-%d") }}" disabled />
+      <input type="hidden" name="waiver_date" id="waiver_date" value="{{ now()|datetime("%Y-%m-%d") }}" />
```

Comment:
Instead of having a ``disabled`` form field and then a separate ``hidden`` form field, you could instead just have a single ``readonly`` form field.  Not a big deal though, it'll work either way.

Reply:
It's easy to miss (and I should probably comment it) but the disabled field displays the _localized_ date and the hidden field is there to store the UTC date! Without access to the page handler for either the prereg page or the transfer badge page, this was the best I could come up with... though that was partly because I ran into errors when trying to add a server default to the field, and I eventually discovered those errors were because we shadow `utcnow()` and I can get around them.

So perhaps the better approach is to go back and add a server default of datetime.utcnow().date() and just get rid of the hidden field? It might be weird to have the form you're submitting not actually control the data, though. Annnnd I should probably ask our lawyer if our attendees should be able to see the waiver and their signature after they preregister...

Reply:
Ohhh, right.  That makes sense.  I would just add a comment then explaining that and leave it as-is.

I am not a lawyer, but I wouldn't worry about making the wavier visible after the fact - we'll be happy to provide a copy if asked.  (In general, we don't hand out paper copies of the on-site waivers we have people sign, though we would certainly provide someone with a copy if they wanted one.)